pysqldb3/README.md

```diff
@@ -299,7 +297,7 @@ Runs query from input SQL string, calls Query object.
  - **`timeme` bool, default True**: If False overrides default behavior that automatically prints query durration time
  - **`no_comment` bool, default False**: If True overrides default behavior to automatically generate a comment on any tables created in query (Postgres only)
  - **`comment` str, default ''**: If provided, appends to automatic table generation comment
- - **`lock_table` str, default None**: ??? Table schema and name to be locked in formate `'schema.table'`
+ - **`lock_table` str, default None**: ??? Table schema and name to be locked in format `'schema.table'`
```

Comment:
A little more detail here would be helpful. If you follow this param through to the query class it is used for creating exclusive locks on the table. See line 106 in query.py